build_docs.py

```diff
@@ -0,0 +1,96 @@
+import os
+import shutil
+import subprocess
+import sys
+from pathlib import Path
+
+REDIRECT_HTML = """
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Redirecting...</title>
+<meta http-equiv="refresh" content="0; URL=./{}/">
+"""
+GITHUB_REPO = os.getenv('GITHUB_REPO')
```

Comment:
make sure to run `black`, as we are mixing single and double quotes in this file which is a style concern.